src/libKinDyn_wrapper/CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+# Copyright (C) 2019 Fondazione Istituto Italiano di Tecnologia (IIT)
+# All Rights Reserved.
+# Authors: Giulio Romualdi <giulio.romualdi@iit.it>
+
+# set target name
+if(WALKING_CONTROLLERS_COMPILE_libKinDyn_wrapper)
+
+  set(LIBRARY_TARGET_NAME KinDyn_wrapper)
+
+  option(ENABLE_RPATH "Enable RPATH for this library" ON)
+  mark_as_advanced(ENABLE_RPATH)
+  include(AddInstallRPATHSupport)
+  add_install_rpath_support(BIN_DIRS "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_BINDIR}"
+    LIB_DIRS "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}"
+    INSTALL_NAME_DIR "${CMAKE_INSTALL_PREFIX}"
+    DEPENDS ENABLE_RPATH
+    USE_LINK_PATH)
```

Comment:
I think you nee to do this only once, no need to duplicate it multiple times.

Reply:
Fixed